server/services/premisesService.ts

```diff
@@ -34,6 +34,15 @@ export default class PremisesService {
     return { name: premises.name, summaryList }
   }
 
+  async getPremisesRadioList(): Promise<Array<{ text: string; value: string }>> {
+    const premisesClient = this.premisesClientFactory(this.token)
+    const premises = await premisesClient.getAllPremises()
+
+    return premises.map(p => {
```

Comment:
Could we order these alphabetically?

Reply:
Sure, I think ideally later down the line they would be sorted by region

Reply:
👍